arch/arm/soc/nordic_nrf5/nrf51/Kconfig.series

```diff
@@ -16,8 +16,6 @@ config  SOC_SERIES_NRF51X
 	select SYS_POWER_LOW_POWER_STATE_SUPPORTED
 	select XIP
 	select HAS_CMSIS
-	select HAS_NORDIC_MDK
```

Comment:
The commit message should mention removing the HAL/MDK

Reply:
You mean "Replace HAL and MDK..." in the subject line is not enough? Should I add something like "The old HAL and MDK have been removed from the source tree." in the message body?

Reply:
When I do something like this, I usually like to do it in three steps: 1) add the new thing 2) rework  plumbing to use the new thing 3) remove the old thing

So, when I first read the commit message, I thought you were just doing 2)